addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java

```diff
@@ -129,4 +153,22 @@ public NAThermostatDataBody getThermostatsDataBody(String equipmentId) {
         return null;
     }
 
+    public NAWelcomeHomeData getWelcomeDataBody(String homeId) {
+        if (getWelcomeApi() != null) {
+            try {
+                return getWelcomeApi().gethomedata(homeId, null).getBody();
+            } catch (Exception e) {
+                logger.warn("An error occured while calling welcome API : {}", e.getMessage());
+            }
+        }
+        return null;
+    }
+
+    public int getWelcomeEventThings() {
+        return configuration.welcomeEventThings;
+    }
+
+    public int getWelcomeUnknownPersonThings() {
```

Comment:
Do you mean Device Configuration? When yes because of NetatmoThingConfiguration which holds equipmentID